src/fetchApi.js

```diff
@@ -0,0 +1,34 @@
+const fetchApi = {
+  getTravelers() {
```

Comment:
you can probably dry up the code for all these GETs by having one method with an endpoint parameter...you can take 'travelers', 'trips', 'destinations', or 'travelers/${travelerID}` as arguments and interpolate them into the url string